apps/api/src/persistence/repositories/__tests__/TestRepositoryProvider.ts

```diff
@@ -45,7 +48,7 @@ export default class TestRepositoryProvider extends RepositoryProvider {
     }
 
     public async deleteAllEdges(): Promise<void> {
-        await this.databaseProvider.getDBInstance().deleteAll(arangoEdgeCollectionID);
+        await this.databaseProvider.getDBInstance().deleteAll(edgeConnectionCollectionID);
```

Comment:
Don't we need to cleanup the other edge collections as well? Did we cleanup all entity collections too?

Reply:
Fixed.